uk.co.compucorp.civicrm.hrcore/CRM/HRCore/Upgrader/Steps/1020.php

```diff
@@ -0,0 +1,63 @@
+
+<?php
+
+trait CRM_HRCore_Upgrader_Steps_1020 {
+
+  /**
+   * Removes duplicated "Localise CiviHR" menu item
+   */
+  public function upgrade_1020() {
+    $this->up1020_deleteLocaliseCiviHRMenuItem();
+    $this->up1020_createLocaliseCiviCRMMenuItemIfItDoesNotExist();
+
+    return TRUE;
+  }
+
+  private function up1020_deleteLocaliseCiviHRMenuItem() {
+    civicrm_api3('Navigation', 'get', [
+      'sequential' => 1,
+      'label' => "Localise CiviHR",
+      'api.Navigation.delete' => ['id' => "\$value.id"],
+    ]);
+  }
+
+  private function up1020_createLocaliseCiviCRMMenuItemIfItDoesNotExist() {
+    $domain = CRM_Core_Config::domainID();
+    $params = ['return' => 'id', 'name' => 'Administer', 'domain_id' => $domain];
+    $administerId = (int) civicrm_api3('Navigation', 'getvalue', $params);
+
+    $params = [
+      'name' => 'Localise CiviCRM',
+      'label' => 'Localise CiviCRM',
+      'permission' => 'access CiviCRM',
+      'parent_id' =>  $administerId,
+      'url' => 'civicrm/admin/setting/localization?reset=1'
+    ];
+
+    $result = civicrm_api3('Navigation', 'get', $params);
+
+    if ($result['count'] == 0) {
+      $params['is_active'] = 1;
+      $newItem = civicrm_api3('Navigation', 'create', $params);
+
+      $firstAdministerChild = $this->up1020_getFirstChildOfParent($administerId);
+      // makes Localise CiviCRM the first
+      // CiviCRM does not allow to set the weight when creating a new item, so
+      // we need to update that after it has been created
+      civicrm_api3('Navigation', 'create', [
+        'id' => $newItem['id'],
+        'weight' => ((int)$firstAdministerChild['weight']) - 1
+      ]);
+    }
+  }
+
+  private function up1020_getFirstChildOfParent($parentID) {
+    $result = civicrm_api3('Navigation', 'get', [
+      'sequential' => 1,
+      'parent_id' => $parentID,
+      'options' => ['sort' => 'weight', 'limit' => 1],
+    ])['values'];
+
+    return array_shift($result);
+  }
```

Comment:
I guess `civilint` would complain about the lack of an empty line before the closing bracket

Reply:
You were right. Civilint is happy now: 
![2018-06-25-084825_861x127_scrot](https://user-images.githubusercontent.com/388373/41848670-9275f0be-7854-11e8-82ad-bd6a39b56990.png)